frontend/components/Editor.js

```diff
@@ -588,31 +586,12 @@ export class Editor extends Component {
                 if (cell_ids.length > 0) {
                     await update_notebook((notebook) => {
                         for (let cell_id of cell_ids) {
-                            if (this.state.cell_inputs_local[cell_id]) {
+                            if (this.state.cell_inputs_local[cell_id] != null) {
                                 notebook.cell_inputs[cell_id].code = this.state.cell_inputs_local[cell_id].code
+                                notebook.cell_inputs[cell_id].run_requested_timestamp = Date.now() / 1000
```

Comment:
Maybe a dumb concern, but what if multiple clients are in different timezones/have different browser times?

In #2296, the cell code at time `T` can be represented by an integer (the number of updates applied since the start) which I planned to use for the "draft" state by comparing with the editor's current version:

https://github.com/fonsp/Pluto.jl/blob/fee51f7c91a8eb9ad2311e9172c8c7228a01ae14/src/webserver/Dynamic.jl#L116

We could add `last_run_version::Int` and `run_requested_version::Int` in this PR? `run_multiple_cells` becomes `run_requested_version = last_run_version + 1` without the Operational transforms updates currently (not sure about conflicts). With operation transform it is updated to be the latest synced version:

https://github.com/fonsp/Pluto.jl/blob/fee51f7c91a8eb9ad2311e9172c8c7228a01ae14/src/webserver/Dynamic.jl#L497

Reply:
Oh doing this with OT would be super nice!!

Right now there is a visual glitch if you collaborate, and first I run a cell, and then you run the same cell, but your clock is 60 seconds behind mine, then your "requested timestamp" will actually be less than the "last run timestamp". It will still run, but not display as "queued" in the frontend.